_data/guidesnavswanlake.yml

```
@@ -39,6 +39,9 @@
     - title: Writing Tests
       url: /learn/testing-ballerina-code/writing-tests/
       active: writing-tests
+    - title: Structuring the tests
```

```suggestion
    - title: Structuring the Tests
```
Please use title case for all titles